### PR TITLE
Add Aggregate by to what's new

### PR DESCRIPTION
**Pull request by 09jvilla**
Users must request enabling this feature in Grafana Cloud via support ticket.

### CODE REVIEW

**Comment by 09jvilla** [comment]
For on-prem users, this is available in both open source and Enterprise. 

Users must have Grafana version 10.2 or later. They must also set a specific feature toggle `metricsSummary=True`.


Their Tempo datasource must also point to a Tempo or Grafana Enterprise Traces database running version 2.2 or greater that has the metrics-summary-api enabled. See how to do that on this page: 
https://grafana.com/docs/tempo/latest/api_docs/metrics-summary/#configuration

**Comment by 09jvilla** [comment]
@joey-grafana - I left you a stub in there to drop in the video link. Feel free to just commit on top of my branch.

File: docs/sources/whatsnew/whats-new-next/index.md [hunk 1]
```diff
@@ -18,6 +18,23 @@ weight: -37
 
 Welcome to Grafana Cloud! Read on to learn about the newest changes to Grafana Cloud.
 
+## Tempo datasource: "Aggregate By" Search option to compute RED metrics over spans aggregated by attribute
+
+<!-- Joey Tawadrous, Jen Villa -->
+<!-- available in on-prem starting with Grafana 10.2, but also requires Tempo/GET v2.2 or greater -->
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-next/index.md:24]: OSS and Enterprise?

File: docs/sources/whatsnew/whats-new-next/index.md [hunk 2]
```diff
@@ -18,6 +18,23 @@ weight: -37
 
 Welcome to Grafana Cloud! Read on to learn about the newest changes to Grafana Cloud.
 
+## Tempo datasource: "Aggregate By" Search option to compute RED metrics over spans aggregated by attribute
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-next/index.md:21]: Can we shorten this title at all? 
```suggestion
## Tempo data source: "Aggregate By" Search option to compute RED metrics over spans aggregated by attribute
```

File: docs/sources/whatsnew/whats-new-next/index.md [hunk 3]
```diff
@@ -18,6 +18,23 @@ weight: -37
...
+
+<!-- Joey Tawadrous, Jen Villa -->
+<!-- available in on-prem starting with Grafana 10.2, but also requires Tempo/GET v2.2 or greater -->
+
+October 24, 2023
+
+_Experimental in Grafana Cloud_
+
+We've added an "Aggregate By" option to the [TraceQL query editor](https://grafana.com/docs/grafana/next/datasources/tempo/query-editor/traceql-search/#write-traceql-queries-using-search) to leverage Grafana Cloud Traces' [metrics summary API](https://grafana.com/docs/grafana-cloud/monitor-infrastructure/traces/metrics-summary-api/). Users can calculate RED metrics (total span count, percent erroring spans, and latency information) for spans of `kind=server` received in the last hour that match your filter criteria, grouped by whatever attributes you specify.
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-next/index.md:30]: Your link actually works just fine, I'm just using version interpolation to simplify them.

```suggestion
We've added an **Aggregate By** option to the [TraceQL query editor](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/datasources/tempo/query-editor/traceql-search/#write-traceql-queries-using-search) to leverage Grafana Cloud Traces' [metrics summary API](https://grafana.com/docs/grafana-cloud/monitor-infrastructure/traces/metrics-summary-api/). You can calculate RED metrics (total span count, percent erroring spans, and latency information) for spans of `kind=server` received in the last hour that match your filter criteria, grouped by whatever attributes you specify.
```

File: docs/sources/whatsnew/whats-new-next/index.md [hunk 4]
```diff
@@ -18,6 +18,23 @@ weight: -37
...
+
+This feature is disabled by default. To enable it, file a Support Escalation.
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-next/index.md:32]:
```suggestion
This feature is disabled by default. To enable it, file contact Grafana Support.
```

File: docs/sources/whatsnew/whats-new-next/index.md [hunk 5]
```diff
@@ -18,6 +18,23 @@ weight: -37
...
+
+Read more [here](https://grafana.com/docs/grafana/next/datasources/tempo/query-editor/traceql-search/#optional-use-aggregate-by).
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-next/index.md:34]:
```suggestion
For more information, refer to the [documentation](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/datasources/tempo/query-editor/traceql-search/#optional-use-aggregate-by).
```

**Review by imatwawana — COMMENTED** [review]
I've made some small wording edits and left some questions. Once the questions are resolved, this will be good to go!

File: docs/sources/whatsnew/whats-new-next/index.md (hunk 1 above)
Comment (09jvilla) [inline docs/sources/whatsnew/whats-new-next/index.md:24]:
```suggestion
<!-- available in on-prem, both open source Grafana and Grafana Enterprise, starting with Grafana 10.2, but also requires Tempo or Grafana Enterprise Traces (GET) v2.2 or greater -->
```

**Comment by 09jvilla** [comment]
> I've made some small wording edits and left some questions. Once the questions are resolved, this will be good to go!

Changes merged @imatwawana !

**Review by imatwawana — APPROVED** [review]
LGTM!

**Comment by 09jvilla** [comment]
@imatwawana @joey-grafana - I think ready for merge when yall have a chance!

**Pull request merged**